airflow/www/views.py

```diff
@@ -1363,6 +1374,14 @@ def success(self):
                                               confirmed, upstream, downstream,
                                               future, past, State.SUCCESS)
 
+    @staticmethod
+    def _get_log_handler():
```

Comment:
Today I was working on one class that contains a similar piece of code. In the future, we will be able to move it.
https://github.com/apache/airflow/pull/9331/commits/530c93c8ff6140e77a3642efd1e9eff3ac6bdd1d

Reply:
Nice. This is also used in `get_logs_with_metadata()`, but I didn't change it because it also uses `task_log_reader`.